app/models/validations/financial_validations.rb

```diff
@@ -24,29 +24,61 @@ def validate_net_income_uc_proportion(record)
   end
 
   def validate_net_income(record)
-    if record.ecstat1 && record.weekly_net_income
+    if record.ecstat1 && record.hhmemb && record.weekly_net_income
       if record.weekly_net_income > record.applicable_income_range.hard_max
-        hard_max = format_as_currency(record.applicable_income_range.hard_max)
         frequency = record.form.get_question("incfreq", record).label_from_value(record.incfreq).downcase
+        hard_max = format_as_currency(record.applicable_income_range.hard_max)
         record.errors.add(
           :earnings,
           :over_hard_max,
           message: I18n.t("validations.financial.earnings.over_hard_max", hard_max:),
         )
         record.errors.add(
-          :ecstat1,
+          :hhmemb,
           :over_hard_max,
-          message: I18n.t("validations.financial.ecstat.over_hard_max", earnings: format_as_currency(record.earnings), frequency:),
+          message: I18n.t("validations.financial.hhmemb.earnings.over_hard_max", earnings: format_as_currency(record.earnings), frequency:),
         )
+        ecstat_fields = %i[ecstat1 ecstat2 ecstat3 ecstat4 ecstat5 ecstat6 ecstat7 ecstat8]
```

Comment:
Would it make sense to add these errors up to hhmemb? I wonder if this would make sense on BU error summary